tests/integration/test_writes/test_writes.py

```diff
@@ -452,7 +452,6 @@ def test_write_parquet_other_properties(
 @pytest.mark.parametrize(
     "properties",
     [
-        {"write.parquet.row-group-size-bytes": "42"},
```

Comment:
Let me see about adding an explicit test case for writing with an explicit row group size from the table properties.